<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-5R7CBE7" failures="0" tests="1" name="Test" time="13.324" errors="0" timestamp="2024-12-28T21:15:53 IST">
  <testcase classname="PurchaseProduct.BaseClass" name="@BeforeMethod navigationToUrl" time="6.979">
    <failure type="java.lang.AssertionError" message="expected [https://rahulnxttrendz.ccbp.tech/] but found [https://rahulnxttrendz.ccbp.tech/login]">
      <![CDATA[java.lang.AssertionError: expected [https://rahulnxttrendz.ccbp.tech/] but found [https://rahulnxttrendz.ccbp.tech/login]
at PurchaseProduct.BaseClass.navigationToUrl(BaseClass.java:58)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod navigationToUrl -->
  <testcase classname="PurchaseProduct.BaseClass" name="@AfterClass terminate" time="-1.735400753301E9">
    <skipped/>
  </testcase> <!-- @AfterClass terminate -->
  <testcase classname="PurchaseProduct.PyrchaseTestExecutionClass" name="purchaseTheProduct" time="0.0">
    <skipped/>
  </testcase> <!-- purchaseTheProduct -->
</testsuite> <!-- Test -->
